docs/src/main/asciidoc/sqs.adoc

```diff
@@ -1742,9 +1745,6 @@ Following IAM permissions are required by Spring Cloud AWS SQS:
 | To use sqsListener with SimpleMessageListenerContainerFactory you will need to add as well
 | `sqs:GetQueueAttributes`
 
-| To use SqsListener with Sqs name instead of ARN you will need
```

Comment:
I might be missing something, but why remove this?

Don't we need this permission for all use cases except for when the user provides the URL directly?